adafruit_tca9548a.py

```diff
@@ -67,6 +67,7 @@ def try_lock(self):
 
     def unlock(self):
         """Pass thru for unlock."""
+        self.tca.i2c.writeto(self.tca.address, bytes([0]))
```

Comment:
```suggestion
        self.tca.i2c.writeto(self.tca.address, b"\x0")
```

Using a byte literal will avoid creating a list to throw away.

Reply:
Neat! done.